<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideingGameMainWindow</class>
 <widget class="QMainWindow" name="SlideingGameMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>402</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>331</width>
    <height>402</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>331</width>
    <height>402</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SlideingGameMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="coardinatedPushButton" name="coardinatedPushButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="coardinatedPushButton" name="coardinatedPushButton_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newGame_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start New Game</string>
    </property>
   </widget>
   <widget class="coardinatedPushButton" name="coardinatedPushButton_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="coardinatedPushButton" name="coardinatedPushButton_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="coardinatedPushButton" name="coardinatedPushButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="coardinatedPushButton" name="coardinatedPushButton_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="coardinatedPushButton" name="coardinatedPushButton_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="coardinatedPushButton" name="coardinatedPushButton_8">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="coardinatedPushButton" name="coardinatedPushButton_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="coardinatedPushButton" name="coardinatedPushButton_10">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="coardinatedPushButton" name="coardinatedPushButton_11">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>11</string>
    </property>
   </widget>
   <widget class="coardinatedPushButton" name="coardinatedPushButton_12">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="coardinatedPushButton" name="coardinatedPushButton_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>13</string>
    </property>
   </widget>
   <widget class="coardinatedPushButton" name="coardinatedPushButton_14">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>14</string>
    </property>
   </widget>
   <widget class="coardinatedPushButton" name="coardinatedPushButton_15">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
   <zorder>coardinatedPushButton_1</zorder>
   <zorder>newGame_pushButton</zorder>
   <zorder>coardinatedPushButton_4</zorder>
   <zorder>coardinatedPushButton_5</zorder>
   <zorder>coardinatedPushButton_6</zorder>
   <zorder>coardinatedPushButton_7</zorder>
   <zorder>coardinatedPushButton_8</zorder>
   <zorder>coardinatedPushButton_9</zorder>
   <zorder>coardinatedPushButton_10</zorder>
   <zorder>coardinatedPushButton_11</zorder>
   <zorder>coardinatedPushButton_12</zorder>
   <zorder>coardinatedPushButton_13</zorder>
   <zorder>coardinatedPushButton_14</zorder>
   <zorder>coardinatedPushButton_15</zorder>
   <zorder>coardinatedPushButton_2</zorder>
   <zorder>coardinatedPushButton_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>coardinatedPushButton</class>
   <extends>QPushButton</extends>
   <header>coardinatedpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
